src/client/rsg-components/Props/renderDefault.tsx

```diff
@@ -9,18 +9,19 @@ export default function renderDefault(prop: PropDescriptorWithFlow): React.React
 	// Workaround for issue https://github.com/reactjs/react-docgen/issues/221
 	// If prop has defaultValue it can not be required
 	if (prop.defaultValue) {
+		const defaultValueString = typeof prop.defaultValue.value === 'string' ? showSpaces(unquote(prop.defaultValue.value)) : showSpaces(String(prop.defaultValue.value));
```

Comment:
I guess we don't really need a condition here?

```suggestion
		const defaultValue = showSpaces(unquote(String(prop.defaultValue.value)));
```

Reply:
Yeah that works too. Although I just realized if the defaultValue isn't a primitive type like an object it would probably show `[object Object]` as the default prop?